packages/select-menu/src/index.tsx

```diff
@@ -193,91 +127,94 @@ class SelectMenuC<T extends OptionBase> extends React.Component<
       ...popoverProps
     }: Partial<PopoverProps> = this.props.popoverProps || {}
 
+    const popoverPropsForAppearance = {
+      accordion: {
+        appearance: 'accordion',
+        matchTargetWidth: true,
+        transitionType: 'expand',
+        targetOffset: -1
+      } as Partial<PopoverProps>,
+      card: {}
+    }[this.props.popoverAppearance || 'accordion']
+
     return (
       <S.Popover
+        {...popoverPropsForAppearance}
         invalid={this.props.invalid}
-        appearance={this.props.appearance}
-        matchTargetWidth={this.props.appearance === 'outline'}
+        minWidth={150}
+        buttonAppearance={this.props.appearance}
         onOpenStarted={() => this.scrollToSelectedItem()}
-        targetOffset={this.props.appearance === 'outline' ? 0 : undefined}
-        transitionType={
-          this.props.appearance === 'outline' ? 'expand' : 'scale'
-        }
         content={({close}) => {
           return (
-            <S.PopoverHost>
+            <React.Fragment>
               {this.props.hasTitle && (
```

Comment:
Maybe it should be `this.props.hasTitle || this.props.hasTitle hasCloseButton`? It seems confusing to me, because when I pass `hasTitle=false` and `hasCloseButton=true` I won't see close button as expect.